microcosm_logging/factories.py

```diff
@@ -10,6 +10,7 @@
 
 @defaults(
     default_format="%(asctime)s - %(name)-12s - [%(levelname)s] - %(message)s",
+    format_string="{asctime} - {name} - [{levelname}] - {message}",
```

Comment:
We shouldn't need any formatters unless we've configured them to be used. The Python defaults won't make any assumptions about a formatter existing named "default"